pkg/vsphere/session/session.go

```diff
@@ -371,6 +379,7 @@ func (s *Session) Populate(ctx context.Context) (*Session, error) {
 			op.Debugf("Cached folders: %s", s.DatacenterPath)
 		}
 		s.VMFolder = folders.VmFolder
+		s.VCHFolder = folders.VmFolder
```

Comment:
So it's here that I would interrogate the config and if no VCHFolder was provided then set it to the `Vm.Folder`